patterns/1-initial/walk-the-innersource-talk.md

```diff
@@ -0,0 +1,141 @@
+how it’s done.
+
+## Problem
+
+The team behind the InnerSource initiative promotes transparency, collaboration, and best practices across the
+organization. However, if the team itself does not adhere to these principles, others may perceive InnerSource as mere
+rhetoric rather than a transformative practice. Without leading by example, adoption and trust in InnerSource
+initiatives may suffer.
+
+## Story (optional)
+
+At Siemens, when the InnerSource journey began, many of the early contributors were deeply inspired by the Open Source
+culture, several were even active participants in community projects. For them, setting up an InnerSource platform
+wasn’t a stretch; it was a natural continuation of how they already worked: asynchronously across locations, sharing
+ideas through issues, and documenting decisions transparently.
+
+This Open Source-inspired mindset gave the InnerSource platform a distinctly different character - transparent,
+collaborative, and developer-friendly - compared to the closed, proprietary environments many developers were used to.
+Unsurprisingly, it quickly attracted experts and engineers seeking more open, cross-team collaboration and meaningful
+technical exchange.
+
+## Context
+
+In many large organizations, IT services rely on a “TicketOps” model: users submit requests into a system, but the
+people and decisions behind the process remain invisible. This lack of transparency often breeds frustration and
+mistrust, making true collaboration across departments difficult.
+
+In contrast, developer-centric initiatives like InnerSource thrive on openness and shared ownership. Instead of hiding
+behind tickets, teams work in open repositories, discuss decisions publicly, and empower others to contribute. This
+shift fosters trust, accelerates problem-solving, and turns support into collaboration.
+
+## Forces
+
+### What makes the problem difficult?
+
+- In large, traditionally structured organizations, teams - especially platform, architecture, or governance groups -
+are often used to working behind closed doors. Decision-making happens in meetings, documentation is internal or
+scattered, and ownership is unclear. Shifting to transparent, open collaboration requires both cultural change and a
+reevaluation of existing workflows.
+- People in central roles may fear scrutiny, overload from unsolicited input, or loss of control. Additionally, moving
+towards open collaboration demands more discipline in documentation, async communication, and community
+management - skills not always prioritized in internal tooling teams.
+
+### What are the trade-offs?
+
+- **Transparency vs. Control**: sharing decisions and code openly invites feedback and builds trust, but also requires a
+mindset shift—accepting discussion, disagreement, and the need to justify choices in public forums.
+
+- **Speed vs. Inclusiveness**: working openly and asynchronously might initially feel slower than making decisions in
```

Comment:
At least for code, I think it's also Speed vs. Quality sometimes. People self-merging and YOLOing quick fixes in private repos vs. getting proper feedback etc. Not sure if we could capture that somewhere here.